highfidelity/hifi
# Remove extra textfield fro address bar input

https://highfidelity.fogbugz.com/f/cases/14287/

Test plan:
- open Goto app in HMD mode
- make sure VK raised
- make sure text can be input to input field using VK
